Application/EdFi.Ods.Admin.Api/Program.cs

```diff
@@ -6,47 +6,63 @@
 using EdFi.Ods.Admin.Api.Infrastructure;
 using log4net;
 
-var builder = WebApplication.CreateBuilder(args);
+namespace EdFi.Ods.Admin.Api
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
```

Comment:
Added a namespace and `Program.Main` and `Program.StartApplication` so that I could call this from the test project.